draft-ietf-scitt-architecture.md

```diff
@@ -155,7 +155,7 @@ Issuers can register their Signed Statements on any Transparency Service, with t
 
 # Introduction
 
-This document describes the scalable, flexible, and decentralized SCITT architecture.
+This document describes the generic, interoperable and scalable SCITT architecture.
```

Comment:
I could not determine if [RFC&322](https://www.rfc-editor.org/rfc/rfc7322#section-3.2) is the most current, but it recommends comma before last element in series.

Overall, I still support this change.

```suggestion
This document describes the generic, interoperable, and scalable SCITT architecture.
```